lib/xnvme_be_vfio_async.c

```diff
@@ -150,6 +150,34 @@ xnvme_be_vfio_async_cmd_io(struct xnvme_cmd_ctx *ctx, void *dbuf, size_t dbuf_nb
 	return -EINVAL;
 }
 
+int
+xnvme_be_vfio_queue_get_completion_fd(struct xnvme_queue *queue)
```

Comment:
@birkelund would this be the correct way of enabling eventfd() notification for completion-notification on the ioqpair? And can the ``q->id`` be utilized as done below?

Reply:
See above. efds should be sized according to number of vectors supported by the device. Not according to the number of queues supported.